import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/ImportAdapter.java

```diff
@@ -110,14 +122,39 @@ public static void main(String[] args) throws IOException, GeneralSecurityExcept
         new DicomWebClient(requestFactory, cstoreDicomwebAddr, cstoreDicomwebStowPath);
     }
 
-    Map<DestinationFilter, IDicomWebClient> destinationMap = configureDestinationMap(
-        flags.destinationConfigInline, flags.destinationConfigPath, credentials);
+    Pair<ImmutableList<Pair<DestinationFilter, IDicomWebClient>>,
+         ImmutableList<Pair<DestinationFilter, AetDictionary.Aet>>> multipleDestinations = configureMultipleDestinationTypesMap(
+        flags.destinationConfigInline, flags.destinationConfigPath, null, credentials);
 
     BackupUploadService backupUploadService = configureBackupUploadService(flags);
 
     DicomRedactor redactor = configureRedactor(flags);
+
```

Comment:
Let's throw an error if sendToAllMatchingDestinations is true but the backup service is null. Since we require the backup service to send to multiple destinations